enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/ConfigHolder.java

```diff
@@ -504,13 +515,60 @@ private void populateJWTGeneratorConfigurations(JWTGenerator jwtGenerator) {
         jwtConfigurationDto.setEnableUserClaims(jwtGenerator.getEnableUserClaims());
         jwtConfigurationDto.setGatewayJWTGeneratorImpl(jwtGenerator.getGatewayGeneratorImpl());
         jwtConfigurationDto.setTtl(jwtGenerator.getTokenTtl());
+        List<Keypair> keypairs = jwtGenerator.getKeypairsList();
+        Keypair signingKey = getSigningKey(keypairs);
```

Comment:
I guess we should add a comment saying that the validation is done at the adapter level hence there is no possibility of returning null here.